tools/flake8-torch/tests/fixtures/deprecated_from_nn.py

```diff
@@ -0,0 +1,8 @@
+import torch
+torch.nn.UpsamplingNearest2d()
+
+from torch import nn as xx
+xx.UpsamplingNearest2d()
+
+import torch.nn as yy
+yy.UpsamplingNearest2d()
```

Comment:
How about that?
```suggestion
import torch.nn as yy
yy.UpsamplingNearest2d()

import torch.nn.UpsamplingNearest2d as ZZZ
ZZZ()

FOO = ZZZ
FOO()
```

Reply:
`FOO = ZZZ; FOO()` will not be caught currently.
I don't think it's a very common pattern.
We can add that later if needed, may be a bit tricky to implement, but should be doable.